applications/plugins/SofaPython/Binding_BaseContext.cpp

```diff
@@ -366,7 +366,7 @@ static PyObject * BaseContext_getObjects(PyObject * self, PyObject * args)
         }
     }
 
-    sofa::helper::vector< boost::intrusive_ptr<BaseObject> > list;
+    sofa::helper::vector< BaseObject::SPtr > list;
```

Comment:
Well for consistency between header & .cpp I would advocate to write it sptr<BaseObject>. But I agree that this is minor details and we can fix that one file at a time.

Reply:
You're correct.  I figured the original code intended ::SPtr in the first place, and I only wanted to fix the bare minimum.